### DISCUSSION TITLE
feature for reset button

### DISCUSSION THREAD

**Discussion by lucafei** [foxglove/community#1039] [General]
hi, guys, i would like to give a suggestion, that foxglove provide a reset button to reset all data in a layout. The reason is when i play back my data via live connection again, there are still last data, that will cause in data mixture.

**Comment by banisadr** [comment]
@lucafei can you elaborate? Are you disconnecting from a live data source and then connecting to a new one, resetting the robot, or something else? Currently if Foxglove receives a new [`sessionId` from the websocket server](https://github.com/foxglove/ws-protocol/blob/main/docs/spec.md#example), the visualization client should reset the data to prevent this mixure.
